ldk/javascript/packages/create-loop/index.js

```diff
@@ -0,0 +1,239 @@
+      let renderedFilename = squirrellyFilename.replace('.squirrelly', '');
+      if (renderedFilename.endsWith('.ts')) {
+        renderedFilename = filenameWithExtension(renderedFilename);
+      }
+      renderTemplate(
+        path.join(sourceSrcPath, squirrellyFilename),
+        path.join(targetSrcPath, renderedFilename),
+      );
+    });
+
+    const sourceAptitudesPath = path.join(sourceSrcPath, 'aptitudes');
+    const targetAptitudesPath = path.join(targetSrcPath, 'aptitudes');
+    await fs.mkdir(targetAptitudesPath);
+
+    const sourceWhispersPath = path.join(sourceSrcPath, 'whispers');
+    const targetWhispersPath = path.join(targetSrcPath, 'whispers');
+    await fs.mkdir(targetWhispersPath);
+
+    // "search" aptitude create "ui" templates
+    aptitudes.forEach(async (aptitude) => {
+      let aptitudeFilename;
+      let whisperFilename = `${aptitude}Whisper`;
+      // Change first letter to uppercase, e.g. ClipboardWhisper
+      whisperFilename[0] = whisperFilename[0].toUpperCase();
+
+      switch (aptitude) {
+        case 'clipboard':
+          aptitudeFilename = 'clipboardListener';
+          break;
+        case 'filesystem':
+          aptitudeFilename = 'filesystemExample';
+          break;
+        case 'keyboard':
+          aptitudeFilename = 'keyboardListener';
+          break;
+        case 'network':
+          aptitudeFilename = 'networkExample';
+          break;
+        case 'search':
+          aptitudeFilename = 'searchListener';
+          whisperFilename = 'UiWhisper';
+          break;
+        case 'window':
+          aptitudeFilename = 'activeWindowListener';
+          break;
+        default:
+          break;
+      }
+
+      const sourceAptitudePath = path.join(
+        sourceAptitudesPath,
+        aptitude === 'search' ? 'ui' : aptitude,
+      );
+      const targetAptitudePath = path.join(
+        targetAptitudesPath,
+        aptitude === 'search' ? 'ui' : aptitude,
+      );
+      await fs.mkdir(targetAptitudePath);
+
+      // Render aptitude
+      renderTemplate(
+        path.join(sourceAptitudePath, `${aptitudeFilename}.ts.squirrelly`),
+        path.join(targetAptitudePath, filenameWithExtension(aptitudeFilename)),
+      );
+
+      // Render aptitude unit test
+      renderTemplate(
+        path.join(sourceAptitudePath, `${aptitudeFilename}.test.ts.squirrelly`),
+        path.join(targetAptitudePath, filenameWithExtension(`${aptitudeFilename}.test`)),
+      );
+
+      // Render whisper
+      renderTemplate(
+        path.join(sourceWhispersPath, `${whisperFilename}.ts.squirrelly`),
+        path.join(targetWhispersPath, filenameWithExtension(whisperFilename)),
+      );
+
+      // Render whisper unit test
+      renderTemplate(
+        path.join(sourceWhispersPath, `${whisperFilename}.test.ts.squirrelly`),
+        path.join(targetWhispersPath, filenameWithExtension(`${whisperFilename}.test`)),
+      );
+
+      return installNodeModules();
```

Comment:
i think this is just returning undefined since installNodeModules is not returning anything. Is that intended?

Reply:
gotta love that linter